swap_meet/vendor.py

```diff
@@ -1,2 +1,99 @@
+from swap_meet.item import Item 
+"""
+Module that contains class Vendor with attribute inventory and instance methods: add, remove.
+Other instance methods:
+To get items in inventory by Item object attribute: get_by_id, get_by_category, get_best_by_category, get_newest_by_category 
+To swap items by attribute: swap_best_by_category, , swap_newest_by_category. 
+"""
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+        
+    def add(self, item):
+        self.inventory.append(item)
```

Comment:
Hmm, in which case would `item` not be in `self.inventory`? Please correct me if I'm mistaken, but I think we don't need this conditional-- at least, the tests still pass when we take this out.